### ISSUE TITLE
Speed and ping time are way off

### ISSUE THREAD

**Issue by rvernica** [sivel/speedtest-cli#539]
I read the *Inconsistency* note in the README, but still the speeds are way off:
```
> speedtest-cli
Retrieving speedtest.net configuration...
Testing from ...
Retrieving speedtest.net server list...
Selecting best server based on ping...
Hosted by Coastside Net (Half Moon Bay, CA) [23.80 km]: 1800000.0 ms
Testing download speed................................................................................
Download: 56.08 Mbit/s
Testing upload speed................................................................................................
Upload: 35.37 Mbit/s
```
![speed-test](https://user-images.githubusercontent.com/241185/44483580-57ea8c00-a600-11e8-8017-38e9db29fd7f.png)
So, it a factor of 6-10 off.

Also, the ping `1800000.0 ms` is way out there. I understand that this is due to some connection errors, but still, such a number should not be printer. Maybe it can require a minimum number of successful connections and print an average of those. Or simply print `N/A` instead of making up a number.

I am using an HTTP proxy to access the internet, but it seems that both methods of testing the speed successfully go through the proxy.

**Comment by HighMans** [comment]
I've had the same issues with my upload speed, it's showing basically 0. Despite the consistency notice, the values are way off and I feel like the creator's just using it as a scapegoat instead of actually looking at the root cause.

**Comment by rvernica** [comment]
I totally agree. A 10-20% difference would be acceptable, but a 10 times off difference is totally wrong. We can help fix the code, but he first has to accept that there is something wrong with it.
